### ISSUE TITLE
APS 重点复习课程之深度学习及应用 | min hjemmeside

### ISSUE THREAD

**Issue by tinsir888** [tinsir888/tinsir888.github.io-gittalk#144]
https://tinsir888.github.io/posts/a942a1b8.html 

 Overview Gradient Vanishing/Exploding Gradient Descent  Batch Gradient Descent Stochastic Gradient Descent Mini-batch Gradient Descent  Overfitting+Underfitting Backpropagation Useful Data Argumentat
